aip/0131.md

```diff
@@ -85,6 +85,36 @@ paths:
   - The variable for the resource ID **must** be named `id`. Variables
     representing the ID of parent resources **must** end with `Id`.
 
+#### RAML
+
+Single-resource `GET` methods **must** be specified in accordance with the
+following RAML spec.
+
+```yaml
+/publishers/{publisherId}/books/{bookId}:
+  uriParameters:
+    publisherId:
+      description: |
```

Comment:
Very much a nit, but as these are single-line descriptions there really isn't a need to utilize a multiline description here.

Reply:
It is just long enough that when I put it on the same line that requires a horizontal bar to view it all! D: